contents/blog/gdpr-google-analytics-privacy-shield.md

```diff
@@ -0,0 +1,82 @@
+---
+date: 2022-01-25
+title: A new 'Privacy Shield' won't solve big tech's GDPR problem
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/google-analytics-gdpr.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Privacy", "Product Analytics", "General"]
+---
+
+Ten years ago today, the European Commission published the first draft of the General Data Protection Regulation (GDPR). But, as [the recent ruling in Austria concerning one website's use of Google Analytics](https://isgoogleanalyticsillegal.com/) proves, how those rules are enforced and interpreted is still contested.
+
+There's lots of detail in the full ruling (link needed), but here's a quick summary:
+
+- An Austrian website was found to have exported personal data of a user to Google servers in the US
+- The complaint argued this was a breach as it was subject to demands from local intelligence agencies
+- The Austrian Data Protection Agency agreed it was a breach and the case remains ongoing 
+
+That's a very simplified version of events, but here are three important takeaways:
+
+### 1 . Google was cleared of any wrongdoing (for now)
```

Comment:
Google 'itself' was cleared... I'd say for emphasis and remove (for now), it sounds salacious/speculative